tests_recording/test_api.py

```diff
@@ -33,8 +32,8 @@ def setUp(self):
 
     def check_version_increase(self):
         # change specfile little bit to have there some change
-        specfile_location = os.path.join(self.lp.working_dir, "python-ogr.spec")
-        with open(specfile_location, "r") as myfile:
+        specfile_location = Path(self.lp_working_dir, "python-ogr.spec")
```

Comment:
- I don't see `self.lp_working_dir` defined anywhere. Do you mean `self.lp.working_dir`?
- Why is the ` LocalProject.working_dir` still `str` instead of `Path`?